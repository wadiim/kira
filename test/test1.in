.863...24
2...6..87
415....39
92..1675.
....23.6.
16.54.89.
..2..4...
857.3....
.......75
