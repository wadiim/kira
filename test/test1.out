786391524
239465187
415278639
924816753
578923461
163547892
692754318
857132946
341689275
